### DISCUSSION TITLE
Nodes always in the Background and independent to the edge routing

### DISCUSSION THREAD

**Discussion by mghaaasele** [xyflow/xyflow#4440] [Layouting]
Is it possible that a Node is always in the Background, so you can't put another in the foreground. Additional with no impact to the edge routing?
Independent on the order of adding the nodes to the panel? I'm looking for Node, which is only a frame for the other nodes to group them visually.

**Comment by bcakmakoglu** [comment]
There is a group node but I assume you want to know if a specific node can *not be elevated* if selected, is that it?
You could enforce that by using some css like

```css
.react-flow__node-custom-group.selected {
	z-index: 0 !important;
}
```

Although there is a prop called `elevateNodesOnSelect` that might not be quite what you're looking for as it would disable elevating the z-index of *all* node types when selected.

Alternatively you could just use a Sub Flow approach [like in this example.](https://reactflow.dev/examples/layout/sub-flows)

**Reply by mghaaasele** [reply]
There's only a group in a functional way, not in any relationship to the react-flow. I think you're solution fits to a react-flow group doesn't it? Unfortunately I can't change to Sub Flow or something like this, because there are many dependencys,...
